skills/declarative/Calendar/Calendar/dialogs/ListDialog/ListDialog.dialog

```diff
@@ -122,7 +129,8 @@
         {
           "$kind": "Microsoft.SetProperties",
           "$designer": {
-            "id": "DX1wQt"
+            "id": "DX1wQt",
+            "comment": "List navigation properties:\n\n$pageSize indicates how many items will be displayed on each page of the list.\n\n$pageIndex indicates which page of results is currently being displayed.\n\n$itemIndex ...\n\n$startIndex indicates the index of the item to be shown at the top of the current page. By default, this is calculated by multiplying the current $pageIndex by the $pageSize.\n\n$pageItems indicates the subarray of items from the overall list that are displayed in the current page.\n\n$pageCount indicates the overall number of pages required to show the full list of items. This is calculated by dividing the number of items by the pageSize (with an extra page for any remaining items).\n\n$isFirstTurn indicates whether this is the first turn of the dialog. This is used to modify the bot response appropriately for subsequent turns."
```

Comment:
nit: use Bot or bot consistently (both are used throughout)

Reply:
I shifted to Bot. This is to align with the Composer UI labels for the fields